### DISCUSSION TITLE
AGW or Orc8r NMS Remove Active subscriber sessions - R1.5 or 1.5.1

### DISCUSSION THREAD

**Discussion by pkapatsos-blinq** [magma/magma#7681] [General]
How do we remove subscribers that show sessions still being active when it is not ? The AGW no longer has the enb or CPEs attached yet it showing the sessions in Orc8 . The subscriber does not even exists on the AGW? 
any idea please how to clear this in active subs .
view on Orc8r
![image](https://user-images.githubusercontent.com/61711855/122782784-00bd0800-d27f-11eb-901d-7c9395d3a173.png)
view that sub does not exist on the AGW. 
![image](https://user-images.githubusercontent.com/61711855/122783154-4da0de80-d27f-11eb-9f77-f36b23ac72c3.png)
What can we do to clear these session please t?

**Comment by karthiksubraveti** [comment]
@themarwhal looks like subscriber state isn't getting purged. Do you know what might be happening here.

**Reply by pkapatsos-blinq** [reply]
@adityawasudeo  can you please check as have a trial customer and the subs will not get removed once attached/detached . Here is a view these subs were attached last week for a brief moment and the 458 IMSI it been over 1month,
![image](https://user-images.githubusercontent.com/61711855/128567056-33b583d4-8a40-40ff-8deb-977fd3c6dd1f.png)
Note seems to be Debian AGW r 1.52 with R1.6 orc8r .. I see R1.6 of AGW is not doing this but am still testing . I just want to know how to flush them off before we add more. and setting to INACTIVE is not working either. thanks

**Reply by ymasmoudi** [reply]
Hi @pkapatsos-blinq, this is indeed a bug in magma side.
@themarwhal created an issue with more details here,
https://github.com/magma/magma/issues/8621
